libs/blocks/hero-marquee/hero-marquee.css

```diff
@@ -0,0 +1,581 @@
+
+/* Static type */
+.hero-marquee .norm p:only-child { margin: 0; }
+.hero-marquee .norm :is(h1, h2, h3, h4, h5, h6) { margin: 0 0 var(--spacing-xs) 0; }
+.hero-marquee .norm .action-area { margin-top: var(--spacing-s); }
+.hero-marquee .norm div > *:last-child { margin-bottom: 0; }
+.hero-marquee .norm div *:first-child { margin-top: 0; }
+
+.hero-marquee > .foreground {
+  max-width: var(--grid-container-width);
+  min-width: var(--grid-container-width);
+  margin: 0 auto;
+  display: grid;
+  gap: var(--spacing-m);
+}
+
+html[dir="rtl"] .hero-marquee .foreground {
+  flex-direction: row-reverse;
+}
+
+.hero-marquee > .foreground.fw {
+  width: 100%;
+}
+
+.hero-marquee .foreground .copy {
+  display: grid;
+  gap: var(--spacing-xs);
+}
+
+.hero-marquee hr {
+  width: 100%;
+  border: none;
+  height: 1px;
+  margin: 0;
+}
+
+.hero-marquee .has-divider {
+  margin: var(--spacing-xxs) 0;
+}
+
+.hero-marquee .badge {
+  font-size: var(--type-body-xs-size);
+  line-height: 1em;
+  font-weight: 400;
+  vertical-align: middle;
+  padding: 4px 8px;
+  margin-inline-start: var(--spacing-xs);
+  border-radius: 4px;
+  background-color: #FFEB00;
+  color: var(--text-color);
+  white-space: no-wrap;
+}
+
+/* content hidden */
+.hero-marquee .hidden {
+  visibility: hidden;
+  height: 0;
+}
+
+.hero-marquee .foreground div:empty {
+  display: none;
+}
+
+/* Row Types */
+.hero-marquee .row-list {
+  text-align: left;
+} 
+
+html[dir="rtl"] .hero-marquee .row-list {
+  text-align: right;
+}
+
+.hero-marquee .row-list .row-wrapper {
+  display: table;
+}
+
+.hero-marquee .row-list li {
+  margin-bottom: var(--spacing-xxs);
+} 
+
+/* row-qr */
+.hero-marquee .row-qrcode .row-wrapper {
+  display: flex;
+  gap: var(--spacing-s);
+  margin: 0;
+}
+
+.hero-marquee .row-qrcode .row-wrapper > p {
+  margin: 0;
+}
+
+.hero-marquee .row-qrcode .qr-code-img {
+  display: none;
+}
+
+.hero-marquee .row-qrcode .google-play a,
+.hero-marquee .row-qrcode .app-store a {
+  width: 135px;
+  height: 40px;
+  display: inline-flex;
+  text-indent:-999em;
+}
+
+.hero-marquee .row-qrcode .google-play a {
+  background: url('/libs/img/ui/google-play.svg') no-repeat transparent;
+}
+
+.hero-marquee .row-qrcode .app-store a {
+  background: url('/libs/img/ui/app-store.svg') no-repeat transparent;
+}
+
+.hero-marquee.center .row-qrcode .row-wrapper {
+  justify-content: center;
+}
+
+.hero-marquee.center .row-list .row-wrapper { 
+  margin: 0 auto; 
+}
+
+.hero-marquee .row-list .icon-list {
+  padding-inline-start: var(--spacing-m);
+  margin: 0;
+}
+
+.hero-marquee li.icon-item {
+  position: relative;
+  list-style: none;
+}
+
+.hero-marquee li::marker {
+  padding-inline-start: 40px;
+}
+
+.hero-marquee li.icon-item::marker {
+  color: transparent;
+}
+
+.hero-marquee li.icon-item span.icon {
+  position: absolute;
+  left: -28px;
+}
+
+html[dir="rtl"] .hero-marquee li.icon-item span.icon {
+  left: unset;
+  right: -38px;
```

Comment:
Is this intended to be larger than LTR? I originally thought it may be an icon quirk, but the spacing different is noticeable.

Reply:
🤷 I think it's just a typo, I've made em the same `28` and verified the visual alignment Thanks